config/Dockerfiles/buildah-push.sh

```diff
@@ -0,0 +1,8 @@
+IMAGE=docker.io/${DOCKER_HUB_USERNAME}/${DOCKER_HUB_IMAGE}
+TAG=$1
+TAGGED_IMAGE=${IMAGE}:${TAG}
+
+buildah bud -t ${IMAGE} .
+buildah push --authfile ${docker_creds} ${IMAGE}
```

Comment:
Your irc question was probably related to ${docker_creds}.  My question is where is this defined even in the old version?  I don't see that it would have any value.  

DOCKER_HUB_USERNAME and DOCKER_HUB_IMAGE are defined in env. so they will be available.  but $IMAGE and $TAG won't be unless you assign them to env.IMAGE and env.TAG before calling the script.

Reply:
IMAGE and TAG are defined in the shell script so should be fine. I don't see docker_creds anywhere though, so don't see this script working

Reply:
docker_creds is a credentials file I added to jenkins.  I thought I read that you access those files as environment variables but I could be wrong.

@p3ck in the old version it's $docker-creds but bash doesn't allow "-" in variable names

Reply:
I don't think that happens for free unless you do something like the prepareCredentials function does https://github.com/CentOS-PaaS-SIG/ci-pipeline/blob/master/src/org/centos/pipeline/PipelineUtils.groovy#L506